package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "container-sweep",
+  "version": "1.0.0",
+  "description": "The Official Container Sweep CLI",
+  "bin": {
+    "container-sweep": "./index.js"
+  },
+  "keywords": [],
+  "author": "Austin Nason",
+  "license": "ISC",
+  "dependencies": {
+    "chalk": "^5.0.1",
+    "config": "^3.3.7",
+    "dockerode": "^3.3.1",
+    "express": "^4.18.1",
+    "node-cron": "^3.0.1",
+    "uuid": "^8.3.2",
```

Comment:
express and uuid are not used:
```bash
npm uninstall express uuid
```

Reply:
I removed express but `node-cron` uses `uuid` under the hood so I'd need to have it installed.